Update Spaces Billing (#2075)
huggingface/hub-docs: docs/hub/spaces-gpus.md

Billing on Spaces is based on hardware usage and is computed by the minute: you get charged for every minute the Space runs on the requested hardware, 
regardless of whether the Space is used.

> During a Space's lifecycle, it is only billed when the Space is actually `Starting` and `Running`. This means that there is no cost during build.

Reviewer: ```suggestion
During a Space's lifecycle, it is only billed when the Space is `Starting` or `Running`. This means that there is no cost during build.
```
